cmd/edit-resources.go

```diff
@@ -197,5 +198,13 @@ func init() {
 			VarType:         "string",
 			Mandatory:       false,
 			DefaultValue:    "",
+		},
+		inputField{
+			Name:            "Fixed Last Octet",
+			FlagName:        "fixed_last_octet",
+			FlagDescription: "enforces the agent to bind the resource to a local IP in the format 192.0.2.X",
+			VarType:         "int",
+			Mandatory:       false,
+			DefaultValue:    0,
```

Comment:
shouldn't the default be something like nil instead? or if the default value is not changed it just doesn't send the key? if you send fixed_last_octet=0 it will very much probably fail.

Reply:
@pablosole please check now. My proposal is to use --fixed_last_octet=null to disable the option